test/cpp/util/channelz_sampler.cc

```diff
@@ -0,0 +1,512 @@
+      if (!status.ok()) {
+        gpr_log(GPR_ERROR, "%s",
+                get_server_context.debug_error_string().c_str());
+        return false;
+      }
+      for (const auto& _server : get_server_response.server()) {
+        all_servers.push_back(_server);
+        StoreServerInJson(_server);
+      }
+      if (!get_server_response.end()) {
+        server_start_id = GetServerID(all_servers.back()) + 1;
+      } else {
+        break;
+      }
+    }
+    std::cout << "Number of servers = " << all_servers.size() << std::endl;
+    return true;
+  }
+
+  // Get sockets that belongs to servers
+  // Store sockets for dumping data
+  void GetSocketsOfServers() {
+    for (const auto& _server : all_servers) {
+      std::cout << "Server " << GetServerID(_server) << " listen_socket: ";
+      for (const auto& _socket : _server.listen_socket()) {
+        int64_t so_id = _socket.socket_id();
+        std::cout << so_id << " ";
+        if (CheckID(so_id)) {
+          grpc::channelz::v1::Socket so = GetSocketRPC(so_id);
+          all_sockets.push_back(so);
+          StoreSocketInJson(so);
+        }
+      }
+      std::cout << std::endl;
+    }
+  }
+
+  // Get all top channels, keep querying until getting all
+  // Store channels for dumping data
+  // No need to check id repeating for top channels
+  bool GetTopChannelsRPC() {
+    int64_t channel_start_id = 0;
+    while (true) {
+      GetTopChannelsRequest get_top_channels_request;
+      GetTopChannelsResponse get_top_channels_response;
+      ClientContext get_top_channels_context;
+      get_top_channels_request.set_start_channel_id(channel_start_id);
+      Status status = channelz_stub->GetTopChannels(&get_top_channels_context,
+                                                    get_top_channels_request,
+                                                    &get_top_channels_response);
+      if (!status.ok()) {
+        gpr_log(GPR_ERROR, "%s",
+                get_top_channels_context.debug_error_string().c_str());
+        return false;
+      }
+      for (const auto& _topchannel : get_top_channels_response.channel()) {
+        top_channels.push_back(_topchannel);
+        all_channels.push_back(_topchannel);
+        StoreChannelInJson(_topchannel);
+      }
+      if (!get_top_channels_response.end()) {
+        channel_start_id = GetChannelID(top_channels.back()) + 1;
+      } else {
+        break;
+      }
+    }
+    std::cout << "Number of top channels = " << top_channels.size()
+              << std::endl;
+    return true;
+  }
+
+  // layer traverse for each top channel
+  void TraverseTopChannels() {
+    for (const auto& _topchannel : top_channels) {
+      int tree_depth = 0;
+      std::queue<grpc::channelz::v1::Channel> channel_queue;
+      std::queue<grpc::channelz::v1::Subchannel> subchannel_queue;
+      std::cout << "Tree depth = " << tree_depth << std::endl;
+      GetChannelDescedence(_topchannel, channel_queue, subchannel_queue);
+
```

Comment:
Lets remove unnecessary blank lines (this comment applies throughout the rest of the file)

Reply:
Done. Also removed other blank lines.